File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        String s="";
        int l=columnNumber;
        while(l>=1)
        {
            l--;
            s=(char)(l%26+65)+s;
            l=l/26;
        }
        return s;
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string str="";
        int col=columnNumber;
        while(col>=1)
        {
            col--;
            str=(char)(col%26+65)+str;
            col=col/26;
        }
        return str;
    }
};